### ISSUE TITLE
Request to add First Female Name & First Male Name to ENS Vision Category 

### ISSUE THREAD

**Issue by saminatapia** [Zimtente/ens-collections#589]
Hey Zimtente, 

Hope you well, 

First of all thanks to you and to the whole ENS Vision team for developing such an amazing 
and user friendly ens application. I have seen it grow slowly and gradually, and wishing you all good luck for further development. 

Below is my request 
Kindly Add the below name to First Female Name on ENS Vision 
samina.eth 

Kindly Add below name to First Male Name on Ens Vison 
quresh.eth 

Awaiting your reply, 

Thanks 
Samina.eth

**Comment by Zimtente** [comment]
Hi samina.eth,

thank you for your kind words - really appreciate that! <3
I added your names to your desired categories - they will be up tomorrow (24/08/2022) after our next update!

Greets
zim

**Comment by saminatapia** [comment]
Hey Zimtente, 

Thank your for your email. 

I just checked the listing on https://www.ens.vision/categories/female-firstnames for samina.eth and 
https://www.ens.vision/categories/male-firstnames for quresh.eth - but seems it is not up yet, may be do i 
have to wait for the update ? Not in a rush, just want to make sure, that I got it listed to the correct category. 

Thanks again for all your support, and wait for your reply. 

Cheers!!
samina.eth
